scripts/generate_stone.ps1

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env pwsh
+. $PSScriptRoot/util.ps1
+EnsureCommand python3
+if (!(Get-Command venv/bin/python -ErrorAction SilentlyContinue)) {
+    python3 -m venv venv
+}
+if (!(Get-Command venv/bin/stone -ErrorAction SilentlyContinue)) {
+    venv/bin/pip install stone
```

Comment:
Nice! Happy to see we aren't going to be using the submodule. However, should we use a requirements file here so we can pin a version? If we make a breaking change and don't add support immediately, we wouldn't want this to just stop working right?